File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/step/ClickStepDTO.java
package com.mokatest.platform.demos.domain.ui.dto.step;// 点击操作

import com.mokatest.platform.demos.domain.ui.uiEnum.click.ClickType;
import com.mokatest.platform.demos.domain.ui.dto.ElementDTO;
import lombok.Data;

@Data
public class ClickStepDTO extends StepBaseDTO {
    private String stepType; // 步骤类型
    private ClickType clickType; // 点击类型
    private ElementDTO element; // 元素
    private String optionValue; // 下拉选项值
}

File: frontEnd/src/types/dto/stepDTO/ClickStepDTO.ts
// 点击操作
import {ClickType} from "@/types/enum/click/ClickType";
import {StepBaseDTO} from "@/types/dto/base/StepBaseDTO";
import {ElementDTO} from "@/types/dto/ElementDTO";

export class ClickStepDTO extends StepBaseDTO {
    stepType: string = "CLICK"; // 步骤类型
    clickType: ClickType | string = "SINGLE_CLICK"; // 点击类型
    element: ElementDTO = new ElementDTO(); // 元素
}